platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -20,10 +23,21 @@
 COMPONENT_CPLD = "CPLD"
 
 BIOS_QUERY_VERSION_COMMAND = 'dmidecode -t 11'
```

Comment:
@stephenxs please move it to the BIOS class

Reply:
fixed